runtime/src/builtins.rs

```diff
@@ -12,25 +12,36 @@ pub struct Builtin {
     pub name: String,
     pub id: Pubkey,
     pub process_instruction_with_context: ProcessInstructionWithContext,
+    // compute units to deduct from transaction's compute budget if builtin
+    // does not consume actual units during process_instruction. No builtin
+    // actively consumes units as bpf does (as of v1.16), but they could
```

Comment:
micro-nit: can we say that the program "manually" consumes units? "actively" sounds like they don't consume any units at all
```suggestion
    // manually consumes units as bpf does (as of v1.16), but they could
```